src/components/views/messages/TextualBody.tsx

```diff
@@ -612,6 +612,7 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
         }
 
         let widgets;
+        const id = "mx_EventTile_content_" + mxEvent.getId();
```

Comment:
`getId()` here will actually return the local id but will return the remote id from sync elsewhere:

![image](https://github.com/matrix-org/matrix-react-sdk/assets/7010307/1d473922-e6a0-4c4e-865e-a1e5ea4ed226)
 
 Probably best to use `mxEvent.getTxnId() ?? mxEvent.getId()`